pkg/monitor/worker.go

```diff
@@ -166,12 +167,13 @@ out:
 		}
 
 		newh := time.Now().Hour()
+		newd := time.Now().Day()
 
 		// TODO: later can modify here to poll once per N minutes and re-issue
 		// cached metrics in the remaining minutes
 
 		if sub != nil && sub.Subscription != nil && sub.Subscription.State != api.SubscriptionStateSuspended && sub.Subscription.State != api.SubscriptionStateWarned {
-			mon.workOne(context.Background(), log, v.doc, newh != h)
+			mon.workOne(context.Background(), log, v.doc, newh != h, d != newd)
```

Comment:
Is there going to be a spike of metrics for every cluster emitted on the hour, every hour, and a spike of metrics every day at midnight? Should there be a random time offset set for each cluster to distribute when the metrics are emitted? (If this doesn't matter now, will it matter in the future?)

Reply:
yes, but I'm bit sceptical how big this spike will be. In region with 100 clusters, we will get 100 more time series.
Once cluster goes down for the issue we get more from them on all pods down, etc
But yes, we need to be aware of something like this.